bugsnag-plugin-android-ndk/src/main/assets/include/bugsnag.h

```diff
@@ -36,6 +39,11 @@ void bugsnag_set_user_env(JNIEnv *env, char* id, char* email, char* name);
 void bugsnag_leave_breadcrumb(char *message, bsg_breadcrumb_t type);
 void bugsnag_leave_breadcrumb_env(JNIEnv *env, char *message, bsg_breadcrumb_t type);
 
+void bugsnag_add_on_error(on_error on_error);
+void bugsnag_remove_on_error(on_error on_error);
```

Comment:
Is it confusing to have these be add and remove operations if there's just one allowed? Should we deliberately deviate from the spec here to differentiate the behaviour?

Reply:
Added some additional context on the rationale behind this in the description.